.github/workflows/ci.yml

```diff
@@ -27,7 +27,9 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
-    - run: npm i
-    # - run: npm ci
-    - run: npm run build --if-present
-    # - run: npm test
+    - run: yarn
+    - run: yarn build
+
+    ## Docker build and push (TODO)
+    - name: Docker build and push to xxx
+      run: docker build -t ghcr.io/amphitheatre-app/playground:v0.0.1 .
```

Comment:
Only the tag branch is handled in docker.yml, where the version number uses the value of the tag, cf. https://github.com/amphitheatre-app/amphitheatre/blob/master/.github/workflows/docker.yml